app/routes/overview/components/Pinned.tsx

```diff
@@ -20,18 +18,20 @@ const Pinned = ({ item, url, meta, style }: Props) => {
   const fetching = useAppSelector(
     (state) =>
       state.frontpage.fetching ||
-      (isEvent(item) ? state.events.fetching : state.articles.fetching),
+      (item && isEvent(item) ? state.events.fetching : state.articles.fetching),
   );
 
   return (
     <Flex column style={style} className={styles.pinned}>
-      <h3 className="u-ui-heading">Festet oppslag</h3>
+      <h3 className="u-ui-heading">
+        {item?.pinned === false ? 'Oppslag' : 'Festet oppslag'}
```

Comment:
Not a big fan of boolean comparisons
```suggestion
        {item?.pinned ? 'Festet oppslag' : 'Oppslag'}
```

Any reason for it?

Reply:
Yes, I wanted it to say "Festet oppslag" while loading as we usually will have something pinned. I realise this is a bit unclear, I should probably just do something like ```fetching || item?.pinned ? ...``` instead